src/utils/melos_client.py

```diff
@@ -15,7 +15,10 @@ def get_user_data(self, melos_id):
         }
 
     def is_member(self, ssn):
-        return True
+        if ssn == '196001010101':
+            return False
+        else:
+            return True
```

Comment:
Är det bara ett test med just det personnumret?

Reply:
Yes, we add that special case so that we can test if a person is not a member